lib/numbers.md

```diff
@@ -4,21 +4,25 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 ---
 
 # What are the different kinds of numbers in Ruby?
+There are two primary types of numbers: 'fixnum', which are integers (whole numbers without decimal points), and 'float', which is a number with a decimal point.
 
-Replace me with your response...
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
++ (addition), - (subtraction), * (multiplication), / (division), ** (exponents), % (modulo)
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+On a string, the + operation will combine strings, returning a composite of the strings entered.
+On a number, the + operation will perform addition, returning the resulting calculation.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+It will not work. It is not possible to perform mathematical operations on strings. 
```

Comment:
We don't care, because the question says we are looking to perform a mathematical operation. :)

But I think I know where you're going. Under my last answer, I'd mentioned "Used with strings, * will repeat the preceding string the number of times indicated after the *." Does that work, or is there something I'm missing?